.github/workflows/main.yml

```diff
@@ -65,12 +65,26 @@ jobs:
       # Compilation and registration of the PyCOMTest server dll
       - name: Set up MSVC
         uses: microsoft/setup-msbuild@v2
+
+      # See https://github.com/actions/runner-images/issues/9701
+      # Adapted from https://github.com/actions/runner-images/issues/9873#issuecomment-2139288682
+      - name: Install missing Visual Studio components
+        run: |
```

Comment:
Tempted to rewrite this as a script under `.github/workflows` to get full intellisense and all that.
At which point may as well rewrite as a python script rather than powershell.

Reply:
Done in [`57e157b` (#2582)](https://github.com/mhammond/pywin32/pull/2582/commits/57e157b003b7628a83f17b5e2002a5fdca90e94d)